playground/infrastructure/helper.py

```diff
@@ -236,39 +237,52 @@ def _validate(tag: dict, supported_categories: List[str]) -> bool:
   """
   valid = True
   for field in fields(TagFields):
-    if tag.get(field.default) is None:
+    if field.default not in tag:
       logging.error(
-        "tag doesn't contain %s field: %s \n"
-        "Please, check that this field exists in the beam playground tag."
-        "If you are sure that this field exists in the tag"
-        " check the format of indenting.", field.default, tag.__str__())
+          "tag doesn't contain %s field: %s \n"
+          "Please, check that this field exists in the beam playground tag."
+          "If you are sure that this field exists in the tag"
+          " check the format of indenting.",
+          field.default,
+          tag.__str__())
+      valid = False
+
+    name = tag.get(TagFields.NAME)
+    if name == "":
```

Comment:
```suggestion
    if not name:
```

This would work for `None` and empty string. Maybe that would be useful?

Reply:
AFAIK tihs is the preferred way to test for empty string / None / etc in Python

[1] https://docs.python.org/3/library/stdtypes.html#truth-value-testing
[2] https://stackoverflow.com/questions/9573244/how-to-check-if-the-string-is-empty